### ISSUE TITLE
feat: wine tasting logs

### ISSUE THREAD

**Issue by Xurape** [luisteofilo/estg-esof2-turno-bc#1]
## TL;DR
A fairly simple system to allow users in an event to **log their experience** on a taste from a specific wine by filling a form with various questions on the characteristics of the wine itself.

## Objective
Allow any user participating in an event to keep a record of their taste experiences on the wines of the event and give that wine a general score based on their evaluation. It will have some types of questions:
- "Yes" and "No" questions
- Radio button questions 
- Multiple choice questions
- Scale of 1 to 10 questions
- Long answer questions
- Short answer questions

And in the final section of the form will be a space for the user to type their own observations on the experience.

All the questions are event-based, and there is going to be a "template questions" that will allow the event administrators to create that questions on a event basis.

## Database Model (First model)
![Image of the first database model for this feature](https://i.ibb.co/RCytJS6/1-Modelagem-ESII-drawio.png)

## Initial expected tasks
- [ ] Check to see if the user is on an event
- [ ] Create the form directly to that specific user after tasting the wine
- [ ] Handle the form; 
- [ ] Check for any potencial security failures on the inputs and avoid any errors
- [ ] Make a general wine score average for a specific wine
- [ ] Display a record of all the tastes the user have done

<p align="center">
<b>Group: BC19</b><br/>
<i>Hugo Guimarães - 29142 | João Ferreira  - 29446</i>
</p>
